src/pages/starter-kit/checkout/tax-use-cases.md

```diff
@@ -204,3 +204,42 @@ Here's an example payload showing how the custom attributes from tax classes app
   }
 }
 ```
+
+
+## Propagation of serialized custom attributes from Tax Class entities to Quote/QuoteItem entities
+
+This new feature populates the `custom_attributes` fields of the `Quote` and `Quote_item` entities as following:
+
+- `Quote`: Any serialized custom attributes from the customer tax class are populated into the `custom_attributes` field of this `Quote` object.
+- `Quote_item`: Any serialized custom attributes from the tax class of the product assocaited to an item` are set as `custom_attributes` of this `Quote_item` object.
+
+Once `custom_attributes` are populated, these are subsequently propagated to the `Order` and `Order_item` entities as per correspondance.
+
+In the case of `Quote_item` currently only serialized custom attributes from the tax class of the `product`are being propagated but there are other types of items as well such as `shipping`or `giftwrap` which will be considered for future enhancements.
+
+### Example
+
+Here's an example of how this looks like in case of a `Quote_item` but it's similar for 'quote':
+
+If the following is the tax class information where we'll mainly focus on the `custom_attributes_serializable` field for now:
```

Comment:
I don't understand this. Replace this with a short simple description of what the example does and how the user would use it. Where do they add the custom attributes? Is this this compatible with PaaS and SaaS? etc